_pages/en_US/troubleshooting.txt

```diff
@@ -463,15 +520,19 @@ Once you've done this, continue from [Section I Step 7](installing-boot9strap-(m
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
-<summary><u>(title/import).db does not exist, create them now?" / invalid database</u></summary>
+<summary><u>Error 08: One or more files are missing or malformed!</u></summary>
+
+One or more files that MSET9 needs to run is missing or corrupted. Re-download the [MSET9 ZIP](https://github.com/zoogie/MSET9/releases/latest) and extract it to the root of your SD card, replacing all existing files, then try again.
```

Comment:
not super important but ZIP is capitalized

Reply:
imo file extensions should always be capitalized

Reply:
we always used `.zip` for that

on that note, we should also explicitly specify _not_ to get the source code which is also a ZIP